### DISCUSSION TITLE
Concerning the Translations |

### DISCUSSION THREAD

**Discussion by the1Riddle** [vim/vim#14574] [General]
Hello;
is new translations into different languages apart from the 11, languages or so that are currently available
welcomed?

**Comment by chrisbra** [comment]
Are you talking about help pages? Or what 11 languages are you talking about?

**Reply by the1Riddle** [reply]
yup the pages

**Reply by chrisbra** [reply]
Any translations is welcome. Translated Help pages are however not distributed with Vim and are usually separate projects. But I am happy to set a link to a new translation available at https://www.vim.org/translations.php
